### ISSUE TITLE
Display issue.

### ISSUE THREAD

**Issue by Czernobo** [RisingTechOSS/android#20]
A lot of the time the screen doesn't power up when you try to unlock it. You have to put the phone on auto brightness for the screen to power up.

**Comment by xiaoniaoawa** [comment]
hummmmmmmm
maybe the bright level too low?
The fix: Make the ambient light sensor less aggressive

**Comment by Czernobo** [comment]
@xiaoniaoawa It's an unofficial device. Poco F5 Pro.

**Comment by xiaoniaoawa** [comment]
Unofficial, so report to a person that port this OS to Poco F5 Pro
Also, did you try use another port of this OS? Maybe only broke on this Build.
Thanks!

**Comment by Czernobo** [comment]
@xiaoniaoawa Yes, tried three different trees, and multiple builds. The issue is still there.
